spec/text-editor-component-spec.js

```diff
@@ -1651,6 +1651,106 @@ describe('TextEditorComponent', function () {
     })
   })
 
+  describe('block decorations rendering', function () {
```

Comment:
Thanks for the suggestion, @mostafaeweda. I am not sure we have any convention to follow in this case, so I preferred consistency with other specs over syntax terseness. We could probably update all the other specs in another PR, although I have no strong preference over one syntax or the other, unless `this` needs to be captured.